Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -191,4 +196,31 @@ namespace AudioCommon
 		g_sound_stream->GetMixer()->StopLogDSPAudio();
 		s_audio_dump_start = false;
 	}
+
+	void IncreaseVolume(unsigned short offset)
+	{
+		SConfig::GetInstance().m_IsMuted = false;
+		int& currentVolume = SConfig::GetInstance().m_Volume;
+		currentVolume += offset;
+		if (currentVolume > AUDIO_VOLUME_MAX)
+			currentVolume = AUDIO_VOLUME_MAX;
+		UpdateSoundStream();
+	}
+
+	void DecreaseVolume(unsigned short offset)
+	{
+		SConfig::GetInstance().m_IsMuted = false;
```

Comment:
The audio configuration window will not indicate whether the volume is muted or not. If the volume is muted via hotkey the volume slider will still show a value, and changing the slider will not unmute the audio.

![Audio configuration](http://i.imgur.com/BUA0zRZ.png)


Further improvements:
* A mute/unmute button/icon in the audio configuration.
 * A button will probably not fit the window without reorganizing its content.
 * An icon is in my opinion a better indicator. See image below for a suggestion (muted audio icon borrowed from http://upload.wikimedia.org/wikipedia/commons/3/3f/Mute_Icon.svg).
* Disable volume slider when audio is muted? Unmute audio if volume slider is changed?
* Visual feedback when changing volume or muting/unmuting via hotkeys. Is graphical overlay possible? 

![Audio muted](http://i.imgur.com/Yi8g3mz.png)